### DISCUSSION TITLE
WSLg creates a sluggish windows 11 and games stop working

### DISCUSSION THREAD

**Discussion by liechtir** [microsoft/wslg#613] [Q&A]
When installing WSLg on my up to date Windows 11, all my games from Origin(Electronic Arts) or Steam stop working. Also, the native windows apps feel sluggish and the task manager shows a high interrupt delay going on.
I uninstalled WSLg, WSL and the "Windows Subsystem for linux" Windows Feature entierly, disabeld Hyper V in the Bios but the system still is sluggish and I cannot start any game.
Any way to reset my system to previous state of WSLg installed. I guess this is somewhat related to graphics driver or so. I also reinstalled the NVIDIA driver, no luck so far.

Anything else I can try?

**Comment by onomatopellan** [comment]
It's weird. Something that changes after installing WSL for the first time is that the hypervisor is enabled at boot. It could be some driver incompatible with hyper-v.

Open an elevated Powershell and type the following to disable it:
`bcdedit /set hypervisorlaunchtype off`
and reboot Windows.

**Comment by takase1121** [comment]
Take my words with a pinch of salt - I had this problem a long time ago and forgot how I solved it. This does sound like you enabled the wrong virtualization platform feature. There's a feature when enabled, virtualizes both Windows and WSL. I think that might be the case.
